src/dmd/cppmangle.d

```diff
@@ -1468,6 +1472,28 @@ private final class CppMangleVisitor : Visitor
         prefix_name(parent);
     }
 
+    /**
+     * Write `Dp` (C++11 function parameter pack prefix) if 't' is a TemplateSequenceParameter (T...).
+     *
+     * Params:
+     *   t      = Parameter type
+     *   params = Template parameters of the function
+     */
+    private void checkParamPack(Type t, TemplateParameters* params)
```

Comment:
I don't really like `check` since that signals checking for errors to me. I'm not sure what term would be better though.

Reply:
Usually I would use `handle` in that situation.

Reply:
I changed it to `handle`.